api/agent/hybrid/client.go

```diff
@@ -38,7 +38,7 @@ func NewClient(u string) (agent.DataAccess, error) {
 	if uri.Scheme == "" {
 		uri.Scheme = "http"
 	}
-	host := uri.Scheme + "://" + uri.Host + "/v1/"
+	host := uri.Scheme + "://" + uri.Host + "v1"
```

Comment:
not sure this was tested, this needs the slashes or you get `http://localhost:8080v1`

Reply:
fixed